vulnerabilities/importers/apache_httpd.py

```diff
@@ -49,47 +62,110 @@ def updated_advisories(self):
         # skips processing the response further to avoid duplicate work
 
         if create_etag(data_src=self, url=self.url, etag_key="ETag"):
-            data = fetch_xml(self.url)
-            advisories = to_advisories(data)
+            links = fetch_links(self.url)
+            self.set_api()
+            advisories = []
+            for link in links:
+                data = requests.get(link).json()
+                advisories.append(self.to_advisory(data))
             return self.batch_advisories(advisories)
 
         return []
 
+    def to_advisory(self, data):
+        cve = data["CVE_data_meta"]["ID"]
+        descriptions = data.get("description", {}).get("description_data", [])
```

Comment:
Just curious, does this cascade all the way down. Could you point me to an example where `descriptions == []` .

Reply:
No, currently there is no example where `descriptions == []`.

I ran this code but got nothing.
``` python
descriptions = data.get("description", {}).get("description_data", [])
if len(descriptions) == 0:
    print("empty descriptions : {}".format(cve))
description = None
for desc in descriptions:
    if desc.get("lang") == "eng":
         description = desc.get("value")
    else:
         print("other language : {}".format(cve))
```

Reply:
This is too defensive then, may mask issues, ideally we want to fail loudly. 

Does  `descriptions = data["description"]["description_data"]` work ? If yes then use that, otherwise try 
`descriptions = data["description"].get("description_data", [])`